lib/SILGen/SILGenPattern.cpp

```diff
@@ -2750,7 +2759,7 @@ void SILGenFunction::emitSwitchStmt(SwitchStmt *S) {
       S->getSubjectExpr(), SGFContext::AllowGuaranteedPlusZero);
 
   // Inline constructor for subject.
-  auto subject = ([&]() -> ConsumableManagedValue {
+  auto subject = [&]() -> ConsumableManagedValue {
```

Comment:
Nitpick: This seems less clear to me, since it's not obvious that you're calling the lambda immediately and therefore the `auto` is going to be a `ConsumableManagedValue` rather than a functor.